hibernate-core/src/main/java/org/hibernate/mapping/Index.java

```diff
@@ -191,6 +191,10 @@ public int getColumnSpan() {
 		return columns.iterator();
 	}
 
+	public java.util.Map<Column, String> getColumnOrderMap() {
+		return columnOrderMap;
```

Comment:
@dreab8 well, I thought about it but I decided to stay consistent with UniqueKey. I can change it if you prefer.